api/src/certification/configuration/domain/usecases/import-sco-whitelist.js

```diff
@@ -12,12 +12,30 @@ export const importScoWhitelist = withTransaction(
    */
   async ({ externalIds = [], centerRepository }) => {
     await centerRepository.resetWhitelist();
-    const numberOfUpdatedLines = await centerRepository.addToWhitelistByExternalIds({ externalIds });
+    const updatedExternalIds = await centerRepository.addToWhitelistByExternalIds({ externalIds });
+
+    const uniqueExternalIds = [...new Set(externalIds)];
+    const externalIdsOnError = uniqueExternalIds.filter((externalId) => {
+      return !updatedExternalIds.includes(externalId);
+    });
+
+    if (externalIdsOnError.length > 0) {
+      const indexShiftAndCsvHeader = 2;
```

Comment:
Pinaillage, deplacer cela hors fonction et afficher comme constante pour pas reinstantier toujours la const

```suggestion
      const INDEX_SHIFT_AND_CSV_HEADER = 2;
```